design/ceph/multi-cluster-service.md

```diff
@@ -0,0 +1,74 @@
+# Multi-cluster Service support in Rook
+
+## Use case
+
+As a rook-ceph user, I should be able to setup mirroing across multiple clusters with overlapping networks in order to protect my data in case of any disaster.
```

Comment:
```suggestion
As a rook-ceph user, I should be able to setup mirroring across multiple clusters with overlapping networks in order to protect my data in case of any disaster.
```